flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
you likely need to add `"skip-icons-check": true`

Reply:
done